torch_frame/nn/encoder/stype_encoder.py

```diff
@@ -788,7 +792,7 @@ def __init__(
         stats_list: list[dict[StatType, Any]] | None = None,
         stype: stype | None = None,
         post_module: Module | None = None,
-        na_strategy: NAStrategy | None = None,
+        na_strategy: NAStrategy | None = NAStrategy.MEDIAN_TIMESTAMP,
```

Comment:
Any reason why we make this na strategy as the default one?

Reply:
Because if no `NAStrategy` is specified, the encoder would fail on NaN timestamps. This happens quite often. So instead of raising an error I think it's better to provide a default strategy in the user experience perspective.